### PR TITLE
Fix folder structure display in Tails guide

### PR DESCRIPTION
**Pull request by dennisreimann**
This way the folder structure gets displayed correctly on the docs website.
Here on github it is displayed ok, nevertheless imho this improves it and makes it even clearer.

Currently the structure on the docs site looks like this, which is very confusing:

tails-folder-stricture-current

### CODE REVIEW

File: docs/using-wasabi/WasabiSetupTails.md [hunk 1]
```diff
@@ -86,41 +90,45 @@ Wasabi will show as a normal application in your activities overview menu, ready
...
-&emsp; &emsp; &emsp; &emsp; &emsp; `/Wallets` (wallet files)  
-&emsp; &emsp; &emsp; &emsp; &emsp; `/Blocks` (blocks)  
-&emsp; &emsp; `/watch-only coldstorage B`  
-&emsp; &emsp; &emsp; `/.walletwasabi`  
-&emsp; &emsp; &emsp; &emsp; `/client` (here we save our wallet files, filters and blocks)  
-&emsp; &emsp; &emsp; &emsp; &emsp; `/Wallets` (wallet files)  
-&emsp; &emsp; &emsp; &emsp; &emsp; `/Blocks` (blocks)
+```
+/Persistent
+|__ /bitcoin-0.18.1        # Bitcoin Core launcher folder
```
Comment (yahiheb) [inline docs/using-wasabi/WasabiSetupTails.md]:
```suggestion
|__ /bitcoin-0.18.1                # Bitcoin Core launcher folder
```
Add some spaces for better display

File: docs/using-wasabi/WasabiSetupTails.md [hunk 2]
```diff
@@ -86,41 +90,45 @@ Wasabi will show as a normal application in your activities overview menu, ready
...
+|__ /Bitcoin               # Bitcoin Core data folder
```
Comment (yahiheb) [inline docs/using-wasabi/WasabiSetupTails.md]:
```suggestion
|__ /Bitcoin                       # Bitcoin Core data folder  
```

File: docs/using-wasabi/WasabiSetupTails.md [hunk 3]
```diff
@@ -86,41 +90,45 @@ Wasabi will show as a normal application in your activities overview menu, ready
...
-&emsp; &emsp; `/watch-only coldstorage B`  
-&emsp; &emsp; &emsp; `/.walletwasabi`  
-&emsp; &emsp; &emsp; &emsp; `/client` (here we save our wallet files, filters and blocks)  
-&emsp; &emsp; &emsp; &emsp; &emsp; `/Wallets` (wallet files)  
-&emsp; &emsp; &emsp; &emsp; &emsp; `/Blocks` (blocks)
+```
+/Persistent
+|__ /bitcoin-0.18.1        # Bitcoin Core launcher folder
+|__ /Bitcoin               # Bitcoin Core data folder  
+|__ /Wasabi                # General Wasabi folder
```
Comment (yahiheb) [inline docs/using-wasabi/WasabiSetupTails.md]:
```suggestion
|__ /Wasabi                        # General Wasabi folder
```

File: docs/using-wasabi/WasabiSetupTails.md [hunk 4]
```diff
@@ -86,41 +90,45 @@ Wasabi will show as a normal application in your activities overview menu, ready
...
+    |__ /Wasabi-X.X.X.deb  # Wasabi installer
```
Comment (yahiheb) [inline docs/using-wasabi/WasabiSetupTails.md]:
```suggestion
    |__ /Wasabi-X.X.X.deb          # Wasabi installer
```

File: docs/using-wasabi/WasabiSetupTails.md [hunk 5]
```diff
@@ -86,41 +90,45 @@ Wasabi will show as a normal application in your activities overview menu, ready
...
+    |__ /BitcoinStore      # No need to keep multiple copies of same filters
```
Comment (yahiheb) [inline docs/using-wasabi/WasabiSetupTails.md]:
```suggestion
    |__ /BitcoinStore              # No need to keep multiple copies of same filters
```

File: docs/using-wasabi/WasabiSetupTails.md [hunk 6]
```diff
@@ -86,41 +90,45 @@ Wasabi will show as a normal application in your activities overview menu, ready
...
+    |__ /CoinJoin wallet   
+    |   |__ /.walletwasabi
+    |       |__ /client    # Here we save our wallet files, filters and blocks
```
Comment (yahiheb) [inline docs/using-wasabi/WasabiSetupTails.md]:
```suggestion
    |       |__ /client            # Here we save our wallet files, filters and blocks
```

File: docs/using-wasabi/WasabiSetupTails.md [hunk 7]
```diff
@@ -86,41 +90,45 @@ Wasabi will show as a normal application in your activities overview menu, ready
...
+    |           |__ /Wallets
+    |           |__ /Blocks
+    |__ /watch-only coldstorage A
+    |   |__ /.walletwasabi
+    |       |__ /client    # Here we save our wallet files, filters and blocks
```
Comment (yahiheb) [inline docs/using-wasabi/WasabiSetupTails.md]:
```suggestion
    |       |__ /client            # Here we save our wallet files, filters and blocks
```

File: docs/using-wasabi/WasabiSetupTails.md [hunk 8]
```diff
@@ -86,41 +90,45 @@ Wasabi will show as a normal application in your activities overview menu, ready
...
+    |           |__ /Wallets
+    |           |__ /Blocks
+    |__ /watch-only coldstorage B
+        |__ /.walletwasabi
+            |__ /client    # Here we save our wallet files, filters and blocks
```
Comment (yahiheb) [inline docs/using-wasabi/WasabiSetupTails.md]:
```suggestion
            |__ /client            # Here we save our wallet files, filters and blocks
```

File: docs/using-wasabi/WasabiSetupTails.md [hunk 9]
```diff
@@ -33,10 +33,12 @@ Verify the PGP signature of the downloaded package, the zkSNACKs' PGP key finger
...
 You can now save your `Wasabi-X.X.X.deb` into the persistent storage, which should look like this:
 
-`/Persistent`  
-&emsp; `/bitcoin-0.18.1` (Bitcoin Core launcher folder)  
-&emsp; `/Bitcoin` (Bitcoin Core data folder)  
-&emsp; `/Wasabi-X.X.X.deb` (Wasabi installer)
+```
+/Persistent  
+|__ /bitcoin-0.18.1    # Bitcoin Core launcher folder
+|__ /Bitcoin           # Bitcoin Core data folder
```
Comment (yahiheb) [inline docs/using-wasabi/WasabiSetupTails.md]:
```suggestion
|__ /Bitcoin           # Bitcoin Core data folder
```
I think the extra spaces are unnecessary.

File: docs/using-wasabi/WasabiSetupTails.md [hunk 10]
```diff
@@ -46,12 +48,14 @@ Wasabi saves session files in `/Home/.walletwasabi/client`, you need to mark the
...
-`/Persistent`  
- &emsp; `/bitcoin-0.18.1` (Bitcoin Core launcher folder)  
- &emsp; `/Bitcoin` (Bitcoin Core data folder)  
- &emsp; `/Wasabi-X.X.X.deb` (Wasabi installer)  
- &emsp; `/.walletwasabi`  
- &emsp; &emsp; `/client` (here we save our wallet files, filters and blocks)
+```
+/Persistent
+|__ /bitcoin-0.18.1    # Bitcoin Core launcher folder
+|__ /Bitcoin           # Bitcoin Core data folder
```
Comment (yahiheb) [inline docs/using-wasabi/WasabiSetupTails.md]:
```suggestion
|__ /Bitcoin           # Bitcoin Core data folder
```

File: docs/using-wasabi/WasabiSetupTails.md [hunk 11]
```diff
@@ -86,41 +90,45 @@ Wasabi will show as a normal application in your activities overview menu, ready
...
-&emsp; `/Wasabi-X.X.X.deb` (Wasabi installer)  
-&emsp; `/.walletwasabi`  
-&emsp; &emsp; `/client` (here we save our wallet files, filters and blocks)  
-&emsp; &emsp; &emsp; `/Wallets` (wallet files)  
-&emsp; &emsp; &emsp; `/Blocks` (blocks)  
-&emsp; &emsp; &emsp; `/BitcoinStore` (filters)
+```
+/Persistent
+|__ /bitcoin-0.18.1        # Bitcoin Core launcher folder
+|__ /Bitcoin               # Bitcoin Core data folder
```
Comment (yahiheb) [inline docs/using-wasabi/WasabiSetupTails.md]:
```suggestion
|__ /Bitcoin               # Bitcoin Core data folder
```

File: docs/using-wasabi/WasabiSetupTails.md [hunk 12]
```diff
@@ -86,41 +90,45 @@ Wasabi will show as a normal application in your activities overview menu, ready
...
-&emsp; &emsp; &emsp; &emsp; &emsp; `/Wallets` (wallet files)  
-&emsp; &emsp; &emsp; &emsp; &emsp; `/Blocks` (blocks)
+```
+/Persistent
+|__ /bitcoin-0.18.1        # Bitcoin Core launcher folder
+|__ /Bitcoin               # Bitcoin Core data folder  
+|__ /Wasabi                # General Wasabi folder
+    |__ /Wasabi-X.X.X.deb  # Wasabi installer
+    |__ /BitcoinStore      # No need to keep multiple copies of same filters
+    |__ /CoinJoin wallet
```
Comment (yahiheb) [inline docs/using-wasabi/WasabiSetupTails.md]:
```suggestion
    |__ /CoinJoin wallet
```

File: docs/using-wasabi/WasabiSetupTails.md [hunk 13]
```diff
@@ -46,12 +48,14 @@ Wasabi saves session files in `/Home/.walletwasabi/client`, you need to mark the
...
- &emsp; `/Wasabi-X.X.X.deb` (Wasabi installer)  
- &emsp; `/.walletwasabi`  
- &emsp; &emsp; `/client` (here we save our wallet files, filters and blocks)
+```
+/Persistent
+|__ /bitcoin-0.18.1    # Bitcoin Core launcher folder
+|__ /Bitcoin           # Bitcoin Core data folder  
+|__ /Wasabi-X.X.X.deb  # Wasabi installer
+|__ /.walletwasabi
+    |__ /client        # wallet files, filters and blocks
```
Comment (yahiheb) [inline docs/using-wasabi/WasabiSetupTails.md]:
```suggestion
    |__ /client        # Here we save our wallet files, filters and blocks
```

File: docs/using-wasabi/WasabiSetupTails.md [hunk 14]
```diff
@@ -86,41 +90,45 @@ Wasabi will show as a normal application in your activities overview menu, ready
...
-&emsp; &emsp; &emsp; `/Wallets` (wallet files)  
-&emsp; &emsp; &emsp; `/Blocks` (blocks)  
-&emsp; &emsp; &emsp; `/BitcoinStore` (filters)
+```
+/Persistent
+|__ /bitcoin-0.18.1        # Bitcoin Core launcher folder
+|__ /Bitcoin               # Bitcoin Core data folder  
+|__ /Wasabi-X.X.X.deb      # Wasabi installer
+|__ /.walletwasabi
+    |__ /client
```
Comment (yahiheb) [inline docs/using-wasabi/WasabiSetupTails.md]:
```suggestion
    |__ /client        # Here we save our wallet files, blocks and filters
```

File: docs/using-wasabi/WasabiSetupTails.md [hunk 15]
```diff
@@ -86,41 +90,45 @@ Wasabi will show as a normal application in your activities overview menu, ready
...
+        |__ /Wallets       # wallet files
```
Comment (yahiheb) [inline docs/using-wasabi/WasabiSetupTails.md]:
```suggestion
        |__ /Wallets
```

File: docs/using-wasabi/WasabiSetupTails.md [hunk 16]
```diff
@@ -86,41 +90,45 @@ Wasabi will show as a normal application in your activities overview menu, ready
...
+        |__ /Blocks        # blocks
```
Comment (yahiheb) [inline docs/using-wasabi/WasabiSetupTails.md]:
```suggestion
        |__ /Blocks
```

File: docs/using-wasabi/WasabiSetupTails.md [hunk 17]
```diff
@@ -86,41 +90,45 @@ Wasabi will show as a normal application in your activities overview menu, ready
...
+        |__ /BitcoinStore  # filters
```
Comment (yahiheb) [inline docs/using-wasabi/WasabiSetupTails.md]:
```suggestion
        |__ /BitcoinStore
```

**Comment by dennisreimann** [comment]
@yahiheb Thanks for pointing out these things that needed cleanup. I edited the file directly on GitHub and the editor does not remove trailing whitespaces. I also incorporated some of your other suggestions in the additional commits.

**Comment by dennisreimann** [comment]
This is what it looks like now:

tails-folder-structure-new

**Comment by PulpCattel** [comment]
Thank you @dennisreimann, 
I was looking for a way to fix it, this seems fine, even though I'm a bit sad that there's no way to simply show non-breaking spaces.

**Comment by dennisreimann** [comment]
@PulpCattel well, using the code fences (```) you can display whitespace as they are – I chose the treet structure, because it makes it even clearer imho.

**Comment by PulpCattel** [comment]
Problem with (```) is that it shows a big black box and I feel like could be annoying for someone. I think best solution would be to use screenshots, but it seems so overkill to me.

File: docs/using-wasabi/WasabiSetupTails.md [hunk 18]
```diff
@@ -21,22 +21,24 @@ You need sudo privileges to install Wasabi, at “Tails Greeter” create your a
...
-Download for Debian/Ubuntu from:  
-http://wasabiukrxmkdgve5kynjztuovbg43uxcbcxn6y2okcrsg7gb6jdmbad.onion (tor hidden service)  
-or  
+Download for Debian/Ubuntu from:
+http://wasabiukrxmkdgve5kynjztuovbg43uxcbcxn6y2okcrsg7gb6jdmbad.onion (tor hidden service)
+or
 https://www.wasabiwallet.io/#download
 
-Verify the PGP signature of the downloaded package, the zkSNACKs' PGP key fingerprint is:  
+Verify the PGP signature of the downloaded package, the zkSNACKs' PGP key fingerprint is:
```
Comment (yahiheb) [inline docs/using-wasabi/WasabiSetupTails.md]: I think the trailing white spaces in this line and in lines above it are needed to display the next line in a new line.

**Comment by yahiheb** [comment]
> This is what it looks like now:

It looks great.
The only thing I would suggest is to add some spaces before the comments to not be above `/watch-only coldstorage A`.
And maybe add a comment next to the first `/client` saying `# Here we save our wallet files and blocks`

File: docs/using-wasabi/WasabiSetupTails.md (hunk 18 above)
Comment (dennisreimann) [reply docs/using-wasabi/WasabiSetupTails.md]: I added `<br>`s for that as we should be more explicit about the line breaks and not rely on whitespaces.

**Pull request merged**